### PR TITLE
Defensive for new methods

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

File: src/multi-selector.js [hunk 1]
```diff
@@ -129,16 +129,18 @@ const MultiSelector = React.createClass({
...
-		this.setState({
-			disabled: this.props.disableInput(e.target.value),
-		}, () => {
-			this.inputChange(e.target.value);
-		});
+		if (this.props.disableInput) {
+			this.setState({
+				disabled: this.props.disableInput(e.target.value),
+			}, () => {
+				this.inputChange(e.target.value);
```
Comment (leahjlou) [inline src/multi-selector.js]: don't we still need to trigger the input change if `this.props.disableInput` is falsy?

**Comment by blittle** [comment]
:camel:

File: src/multi-selector.js (hunk 1 above)
Comment (blittle) [inline src/multi-selector.js]: Probably right. Can we add some tests for this?

Comment (blittle) [inline src/multi-selector.js]: Currently the forms experience is broken in prod. This change should fix it :)

Comment (dckesler) [inline src/multi-selector.js]: Yeeeeesssss

**Comment by leahjlou** [comment]
🍉

**Pull request merged**